README.md

```diff
@@ -167,8 +168,9 @@ This auth is disabled by default. Options to enable it:
 | JWTFROMEXTRAPROVIDER_PROVIDER_URL | "" | Set to the oidc provider url
 | JWTFROMEXTRAPROVIDER_ISSUER | `` | The issuer |
 | JWTFROMEXTRAPROVIDER_ISSUERNAME | `` | The issuer name |
-| JWTFROMEXTRAPROVIDER_HEADER_NAME | "" | Set to the header name where the JWT is set
+| JWTFROMEXTRAPROVIDER_COOKIE_NAME | "" | Set to the cookie name where the JWT is set
 | JWTFROMEXTRAPROVIDER_CLIENTID | "" | Set to the clientID the jwt should have
+| JWTFROMEXTRAPROVIDER_SETHEADER | "" | Set the HTTP header to the JWT value. Empty to not set any header
```

Comment:
What is this one used for?

Reply:
I've added this one to optionally set a http header on the target cluster with the JWT as well, so we can set "arista-token" as well if we want, in case some backend want to use this.

Reply:
I can remove it if you think that will be useless and it makes things more complicated.